roles/etcd/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Install etcd
+  yum: pkg=etcd state=installed disable_gpg_check=yes
```

Comment:
state should be present not installed. See http://docs.ansible.com/yum_module.html